### PR TITLE
[RHACS] Removed Tech preview note for Scanner V4 instances

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-24917

- Removed Tech preview snippet for Scanner V4 instances.
- Note: One instance remains which is in Release notes 4.4, but that page is replaced by 4.5 release notes, so we can skip that one.

Cherry-pick into `rhacs-docs-4.5`.

Preview:
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/architecture/acs-architecture.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/cli/command-reference/roxctl-scanner.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/acscs-architecture.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/acscs-default-requirements.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/install-secured-cluster-cloud-other.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/acs-default-requirements.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-config-options-ocp.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-ocp.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-secured-cluster-ocp.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_other/install-central-other.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_other/install-secured-cluster-other.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-image-vulnerability-scanners.html
- https://78611--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html

### CODE REVIEW

File: integration/integrate-with-image-vulnerability-scanners.adoc [hunk 1]
```diff
@@ -35,12 +35,7 @@ You can set up {product-title-short} to obtain image vulnerability data from the
...
-+
---
-:FeatureName: Scanner V4
-include::snippets/technology-preview.adoc[]
---
+* Scanner V4: Beginning with {product-title-short} version 4.4, a new scanner is introduced that is built on link:https://github.com/quay/claircore[ClairCore], which also powers the link:https://github.com/quay/clair[Clair] scanner. Scanner V4 supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. For version 4.4, if you enable this scanner, you must also enable the StackRox Scanner. For more information about Scanner V4, including links to the installation documentation, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
```
Comment (tmalove) [inline integration/integrate-with-image-vulnerability-scanners.adoc:38]: From the first preview bullet, this instance of TP should be removed:

image

File: modules/acs-architecture-overview.adoc [hunk 2]
```diff
@@ -17,9 +17,6 @@ However, the underlying components and the interactions between them remain the
 
 The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 is available. Installation of Scanner V4 is optional, but provides additional benefits.
```
Comment (tmalove) [inline modules/acs-architecture-overview.adoc:18]: image

File: modules/acs-central-services-overview.adoc [hunk 3]
```diff
@@ -14,11 +14,6 @@ It handles API interactions and user interface ({product-title-short} Portal) ac
 You can use the same Central instance to secure multiple {ocp} or Kubernetes clusters.
 * *Central DB*: Central DB is the database for {product-title-short} and handles all data persistence. It is currently based on PostgreSQL 13.
 * *Scanner V4* (Technology Preview): Beginning with version 4.4, {product-title-short} contains the Scanner V4 vulnerability scanner for scanning container images. Scanner V4 is built on link:https://github.com/quay/claircore[ClairCore], which also powers the link:https://github.com/quay/clair[Clair] scanner. Scanner V4 supports scanning of language and OS-specific image components. For version 4.4, you must use this scanner in conjunction with the StackRox Scanner to provide node and platform scanning capabilities until Scanner V4 support those capabilities. Scanner V4 contains the Indexer, Matcher, and DB components.
```
Comment (tmalove) [inline modules/acs-central-services-overview.adoc:16]: image

File: modules/acs-secured-cluster-services-overview.adoc [hunk 4]
```diff
@@ -19,11 +19,6 @@ It collects container runtime and network activity information and sends the col
 * *StackRox Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component. However, on {ocp}, {product-title-managed-short} installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
```
Comment (tmalove) [inline modules/acs-secured-cluster-services-overview.adoc:19]: image

**Review by tmalove — APPROVED** [review]
@gaurav-nelson  Check the TP instances to see if they should show...Some of the content was not an exact match in the preview, so I want to have you verify before merging. Thanks!

**Review by tmalove — APPROVED** [review]
@gaurav-nelson  Check the TP instances to see if they should show...Some of the content was not an exact match in the preview, so I want to have you verify before merging. Thanks!

**Comment by gaurav-nelson** [comment]
Thank you, @tmalove, for checking this. I missed removing instances with `... * (Tech...`. I've deleted all remaining instances and checked TP for appearances with Scanner V4.

**Pull request merged**